src/main/java/frc/robot/Constants.java

```diff
@@ -19,11 +24,87 @@ public class Constants {
     public static final double MK4I_STEER_FRICTION_VOLTAGE = 0.25;
     public static final double MK4_WHEEL_RADIUS = Units.inchesToMeters(1.93);
+    public static final double WHEEL_COF = 1.2;
     public static final Slot0Configs MK4I_STEER_PID_CONSTANTS = new Slot0Configs()
             .withKP(50);
 
     public static final double DEBOUNCE_TIME = 0.5;
 
+    public static final double yOffset = 158.5;
+    public static final double xBranchDistanceFromCenter = 31.625;
+    public static final double yBranchDistanceFromCenter = 6.5;
+    public static final Transform2d ROBOT_REEF_OFFSET = new Transform2d(Units.inchesToMeters(21.125), 0, Rotation2d.fromDegrees(180));
+    public static final Transform2d ROBOT_CORAL_STATION_OFFSET = new Transform2d(Units.inchesToMeters(5), 0, Rotation2d.fromDegrees(180));
+
+    public static final Pose2d BLUE_BRANCH_A = getBranchPose(3, false, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_B = getBranchPose(3, true, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_C = getBranchPose(4, false, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_D = getBranchPose(4, true, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_E = getBranchPose(5, false, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_F = getBranchPose(5, true, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_G = getBranchPose(0, false, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_H = getBranchPose(0, true, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_I = getBranchPose(1, false, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_J = getBranchPose(1, true, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_K = getBranchPose(2, false, 176.75, yOffset);
+    public static final Pose2d BLUE_BRANCH_L = getBranchPose(2, true, 176.75, yOffset);
+
+    public static final Pose2d RED_BRANCH_A = getBranchPose(0, true, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_B = getBranchPose(0, false, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_C = getBranchPose(1, true, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_D = getBranchPose(1, false, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_E = getBranchPose(2, true, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_F = getBranchPose(2, false, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_G = getBranchPose(3, true, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_H = getBranchPose(3, false, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_I = getBranchPose(4, true, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_J = getBranchPose(4, false, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_K = getBranchPose(5, true, 514.125, yOffset);
+    public static final Pose2d RED_BRANCH_L = getBranchPose(5, false, 514.125, yOffset);
```

Comment:
The `176.75` and `514.125` constants should be transformed into `public static final` constants, and be in meters.